lms/djangoapps/ccx/views.py

```diff
@@ -208,9 +213,31 @@ def create_ccx(request, course, ccx=None):
         email_params=email_params,
     )
 
+    assign_coach_role_to_ccx(ccx_id, request.user, course.id)
```

Comment:
@pdpinch  It is for the user who already have created ccx.
if user dont have coach right on his ccx but he has coach rights on master course then when he opens his dashboard from master course there (L138) I assign him coach role on ccx if the role dont exist.